lib/frizz/remote.rb

```diff
@@ -14,13 +14,26 @@ def files
     end
 
     def upload(file, key, options = {})
+      encoding_option = {}
+
+      if options[:prefer_gzip]
+        return unless key.ends_with? '.gz'
```

Comment:
So if `prefer_gzip` is on, we don't allow uploading any files except those that are gzipped? If that's the case, we should probably rename the option as "prefer" makes me think it'll use the gz if available, otherwise fallback to the regular file